### PR TITLE
Updated a bunch of URLs again.

### PR DESCRIPTION
**Pull request by Peppernrino**

### CODE REVIEW

File: source/layouts/partials/_acquire_details.erb [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
                             <ul style="list-style-type: none;">
                                 <li>
-                                    <a href="https://bisq.network/">
+                                    <a href="https://markets.bisq.network/?market=grc_btc">
                                         <img src="../assets/img/Exchanges/bisq.png" alt="bisq" />
                                     </a>
                                 </li>
                                 <li>
-                                    <a href="https://market.rudex.org/#/asset/RUDEX.GRC">
+                                    <a href="https://staging.rudex.org/#/asset/RUDEX.GRC">
```
Comment (barton2526) [inline source/layouts/partials/_acquire_details.erb]: This is not the correct URL.  markets.rudex.org is correct

**Comment by barton2526** [comment]
Please rebase this to the new cleaned up master branch so it doesn't have 11 commits.  It should only need 5.

**Comment by Peppernrino** [comment]
i can redo this today. sorry about all that. :P

**Comment by Peppernrino** [comment]
done. new pull @ https://github.com/gridcoin-community/Gridcoin-Site/pull/129

**Pull request closed without merging**